gax/src/test/java/com/google/api/gax/batching/v2/BatcherImplTest.java

```diff
@@ -64,11 +64,23 @@
 
   private Batcher<Integer, Integer> underTest;
   private LabeledIntList labeledIntList = new LabeledIntList("Default");
+  private BatchingSettings batchingSettings =
+      BatchingSettings.newBuilder()
+          .setRequestByteThreshold(1000L)
+          .setElementCountThreshold(1000L)
+          .build();
 
   /** The accumulated results in the test are resolved when {@link Batcher#flush()} is called. */
   @Test
   public void testResultsAreResolvedAfterFlush() throws Exception {
-    underTest = new BatcherImpl<>(SQUARER_BATCHING_DESC_V2, callLabeledIntSquarer, labeledIntList);
+    BatchingSettings settings =
+        BatchingSettings.newBuilder()
+            .setElementCountThreshold(null)
+            .setRequestByteThreshold(null)
+            .build();
```

Comment:
Won't this throw an exception for the null params? How is this test even passing now?

Reply:
Have removed this condition as it is now invalid.